rotkehlchen/tests/api/test_metadata.py

```diff
@@ -0,0 +1,60 @@
+from typing import TYPE_CHECKING
+
+import requests
+
+from rotkehlchen.chain.ethereum.airdrops import AIRDROPS
+from rotkehlchen.chain.ethereum.defi.protocols import DEFI_PROTOCOLS
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response_with_result
+from rotkehlchen.types import SupportedBlockchain
+
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
+
+
+def test_metadata_endpoint(rotkehlchen_api_server: 'APIServer') -> None:
+    """Test that the metadata endpoints works properly"""
+    airdrops_expected_result = [
+        {
+            'identifier': identifier,
+            'name': airdrop.name,
+            'icon': airdrop.icon,
+        }
+        for identifier, airdrop in AIRDROPS.items()
+    ]
+
+    defi_expected_result = [
```

Comment:
Can you separate this test with a comment explaining before each test what endpoint is being tested?